README.osx.catalina.md

```diff
@@ -1,58 +0,0 @@
-ENCODE Metadata Database
```

Comment:
Why was this deleted?

Reply:
Are we still supporting mojave and other older versions of mac?

Reply:
This is the README now.  Also having catalina in the file name was confusing for people on mojave.